docs/howtos/DataPlane Generation - DPG/03convenient.mdx

```diff
@@ -174,7 +185,16 @@ FIXME
 <TabItem value="java" label="Java" >
 
 ```java
-FIXME
+package com.petstorenamespace.implementation.models;
```

Comment:
case of internal in Java

model in impl, which would not get exposed to customer

Reply:
nit: add a comment to this code? saying the model is in `impl` package.

Reply:
good point, added.